suite/auto-sync/src/autosync/MCUpdater.py

```diff
@@ -466,10 +469,12 @@ def extract_llvm_mc_cmds(self, cmds: str) -> list[LLVM_MC_Command]:
     def gen_all(self):
         log.info("Check prerequisites")
         disas_tests = self.mc_dir.joinpath(f"Disassembler/{self.arch}")
-        test_paths = [disas_tests]
+        test_paths = [disas_tests, self.mc_dir.joinpath(self.arch)]
         self.check_prerequisites(test_paths)
         log.info("Generate MC regression tests")
-        llvm_mc_cmds = self.run_llvm_lit(test_paths)
+        llvm_mc_cmds = self.run_llvm_lit(
+            [path for path in test_paths if exists_and_is_dir(path)]
```

Comment:
What bug did you run into here?

Reply:
It seems like there is an error when running a step using a path that doesn't exist.